nx-X11/include/Imakefile

```diff
@@ -16,11 +16,8 @@ GLXDIR = GL
 
 SUBDIRS = extensions $(GLXDIR)
 
-AU_HEADERS = Xauth.h
```

Comment:
Please drop this commit where you remove Xauth.h. This header is a reduced form of Xauth.h and references functions in nx-X11/lib/X11/Au*.c. This header file is "ours" and must not be replaced by its X.org pendant..